easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '5.4.1'
+
+homepage = 'http://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale
+materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics,
+from first principles."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+# Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
+sources = [SOURCELOWER_TAR_GZ]
+
+prebuildopts = 'cp arch/makefile.include.linux_intel ./makefile.include && '
+
+buildopts = 'all LIBS="lib" BLACS="$LIBBLACS" '
```

Comment:
it is, but I suppose it's better to use `unset LIBS` in case what's in the makefile changes